src/main/java/data_factories/EventFactory.java

```diff
@@ -0,0 +1,17 @@
+package data_factories;
+
+import data_objects.EventData;
+import org.json.simple.JSONObject;
+
+public class EventFactory {
```

Comment:
misnamed? maybe EnemyDataFactory or just leave out the factory since this isnt using another dependency?